app/models/compute_resource.rb

```diff
@@ -6,6 +6,11 @@ class ComputeResource < ActiveRecord::Base
   include Parameterizable::ByIdName
   encrypts :password
 
+  attr_accessible :name, :provider, :description, :url, :display_type,
```

Comment:
Should some of these be moved into more appropriate subclasses where they're used?